assignments/number-functions/code.js

```diff
@@ -0,0 +1,42 @@
+function add (a,b) {
```

Comment:
Lots of kids seem to have picked up this syntax somewhere. Which is fine: it's legal Javascript. However I'm trying to teach you a slightly different syntax and want you to get comfortable with it.

```suggestion
const add = (a, b) => {
```

Can you convert these functions to that syntax to make sure you're comfortable with it?

Reply:
done, sorry i havent seen this untill today